tests/php/test_class-wc-connect-service-settings-store.php

```diff
@@ -2,26 +2,42 @@
 
 class WP_Test_WC_Connect_Service_Settings_Store extends WC_Unit_Test_Case {
 
-	protected $settings_store;
```

Comment:
For now I think it's fine to cache a copy of the `WC_Connect_Service_Settings_Store` instance for reuse. Is there a reason a new instance is used for each test?